 boom/report.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/boom/report.py b/boom/report.py
index 8e55aa3..f7ff4a1 100644
--- a/boom/report.py
+++ b/boom/report.py
@@ -121,7 +121,7 @@ class BoomReportOpts(object):
 
 class BoomReportObjType(object):
     """BoomReportObjType()
-        Class representing a type of objecct to be reported on.
+        Class representing a type of object to be reported on.
         Instances of ``BoomReportObjType`` must specify an identifier,
         a description, and a data function that will return the correct
         type of object from a compound object containing data objects
