diff -up boom-boot-1.6.8/pyproject.toml.orig boom-boot-1.6.8/pyproject.toml
--- boom-boot-1.6.8/pyproject.toml.orig	2025-11-04 17:39:34.159066461 +0000
+++ boom-boot-1.6.8/pyproject.toml	2025-11-04 17:39:46.440703634 +0000
@@ -26,7 +26,7 @@ Issues = "https://github.com/snapshotman
 
 [build-system]
 requires = [
-    "setuptools>=61.0",
+    "setuptools>=53.0",
 ]
 build-backend = "setuptools.build_meta"
 
